### DISCUSSION TITLE
Clone repository in container volume fails to clone with bre GitHub cli installed

### DISCUSSION THREAD

**Discussion by RClaudiuM** [devcontainers/community#187] [Q&A]
I have GitHub Cli installed on my machine through Brew and i am logged in with two users on it

This is a sample of what i have in my `.gitconfig`

```
[user]
	name = user name
	email = user.email@domain.test
[credential]
	helper = osxkeychain
[push]
	autoSetupRemote = true
[credential "https://github.com"]
	helper = 
	helper = !/opt/homebrew/bin/gh auth git-credential
[credential "https://gist.github.com"]
	helper = 
	helper = !/opt/homebrew/bin/gh auth git-credential
```

When i try to run the `Dev Containers: Clone Repository in Container volume` command from VSCode it tries to clone the repo and i get this output where i am prompted to input my username and password

```
...
[1282 ms] 
[1282 ms] Start: Run in container: # Copy /Users/claudiu.roman/.ssh/known_hosts to /root/.ssh/known_hosts
[1283 ms] 
[1283 ms] 
[1398 ms] Start: Run in container: git clone --depth 1 --branch main https://github.com/RClaudiuM/clone-from-github.git .
Cloning into '.'...
/opt/homebrew/bin/gh auth git-credential get: line 0: /opt/homebrew/bin/gh: not found
Username for 'https://github.com': 
```

after i input something i get this

<img width="743" alt="image" src="https://github.com/user-attachments/assets/257c69ae-5bf0-4260-9fe1-656f88382a74" />

I get from the logs that the container tries to run the github cli that is installed from Brew and it does not find it.

It works as expected if i comment the following lines from my `.gitconfig'

```
[credential "https://github.com"]
	helper = 
	helper = !/opt/homebrew/bin/gh auth git-credential
[credential "https://gist.github.com"]
	helper = 
	helper = !/opt/homebrew/bin/gh auth git-credential
```

My questions are:
- did somebody else stumbled upon this issue?
- if yes, did you managed to fix it?

I'm aware this might not be the best place to post this but i couldn't think of any better place

**Comment by bamurtaugh** [comment]
@chrmarti I'd love to get your thoughts here!

**Reply by RClaudiuM** [reply]
I can provide more logs if needed.
The Docker Development Environments feature does not have this problem. It clones the repo even if it's private.
